humanized_opening_hours/main.py

```diff
@@ -628,13 +628,13 @@ def get_localized_names(self):
             A dict with the keys "days" and "months" containing lists
             of respectively 7 and 12 strings.
         """
+        day_names_items = babel.dates.get_day_names(locale=self.locale).items()
+        month_names_items = babel.dates.get_month_names(locale=self.locale).items()
         return {
-            "days": list(
-                babel.dates.get_day_names(locale=self.locale).values()
-            ),
-            "months": list(
-                babel.dates.get_month_names(locale=self.locale).values()
-            )
+            # names sorted by day index (from 0:Monday to 6:Sunday)
+            "days": list(day_name for _, day_name in sorted(day_names_items)),
```

Comment:
It's quite superficial, but why using `list(day_name for _, day_name in sorted(day_names_items))` and not `[day_name for _, day_name in sorted(day_names_items)]`?

Reply:
Thank you so much for this fix!
Sorry for the late, I was so busy the last days.

Reply:
No problem! Using list comprehension is more pythonic indeed, I updated the PR.

Travis is probably quite easy to set up, but I can understand that's not a priority.